src/components/LoanApplications.jsx

```diff
@@ -0,0 +1,39 @@
+import { Container, Typography } from '@mui/material';
+import React from 'react';
+import { Variables } from '../common/variables';
+import {
+  Chart,
+  PieSeries,
+} from '@devexpress/dx-react-chart-material-ui';
+
+const data = [
+  { argument: 'Monday', value: 10 },
+  { argument: 'Tuesday', value: 40 },
+  { argument: 'Wednesday', value: 10 },
+  { argument: 'Thursday', value: 20 },
+  { argument: 'Friday', value: 20 },
+];
+
+export const LoanApplications = () => {
+
+  return (
+    <Container sx={{
+      margin: 2,
+      borderRadius: 2,
+      padding: 2,
+      border: `0.1rem solid ${Variables.lightGrey}`,
+    }}>
+      <Chart
+        data={data}
+      >
+        <Typography variant='h4' sx={{ px: 1, py: 2, pb: 5 }}>Loan Applications</Typography>
+        < PieSeries valueField="value"
+          argumentField="argument"
+          color="#f4d742"
```

Comment:
Please move the color code to variables.js so it's much to change and reuse

Reply:
Oops, that is actually not supposed to be there, it is not changing the color as expected so I will remove it altogether